app/posts/register-trainee-teachers/2021-04-20-investigating-free-text-responses-from-apply.md

```diff
@@ -0,0 +1,333 @@
+      }
+    ],
+    [
+      {
+        text: "Masters"
+      },
+      {
+        text: "10"
+      }
+    ],
+    [
+      {
+        text: "PGCE"
+      },
+      {
+        text: "9"
+      }
+    ],
+    [
+      {
+        text: "Diploma"
+      },
+      {
+        text: "7"
+      }
+    ],
+    [
+      {
+        text: "Licenciatura"
+      },
+      {
+        text: "6"
+      }
+    ],
+    [
+      {
+        text: "Master degree"
+      },
+      {
+        text: "6"
+      }
+    ],
+    [
+      {
+        text: "Master’s Degree"
+      },
+      {
+        text: "6"
+      }
+    ]
+  ]
+}) }}
+
+We can see that most of the remaining free text answers are where the user did not provide enough detail - or perhaps entered a qualification that was not a degree. Where the applicant’s qualification is not a degree it’s unlikely the record would end up in Register - as they would likely not be eligible for the course (for courses that require a degree).For the remaining ones we may need providers to amend the record to give the detail required.
+
+{{ govukTable({
+  caption: "Top free text entries for degree subject",
+  captionClasses: "govuk-table__caption--m",
+  firstCellIsHeader: false,
+  head: [
+    {
+      text: "Free text input"
+    },
+    {
+      text: "Frequency"
+    }
+  ],
+  rows: [
+    [
+      {
+        text: "English"
+      },
+      {
+        text: "35"
+      }
+    ],
+    [
+      {
+        text: "Physical Education"
+      },
+      {
+        text: "9"
+      }
+    ],
+    [
+      {
+        text: "English Language and Literature"
+      },
+      {
+        text: "9"
+      }
+    ],
+    [
+      {
+        text: "Accounting and Finance"
+      },
+      {
+        text: "9"
+      }
+    ],
+    [
+      {
+        text: "History and Politics"
+      },
+      {
+        text: "7"
+      }
+    ],
+    [
+      {
+        text: "Sport Science"
+      },
+      {
+        text: "5"
+      }
+    ],
+    [
+      {
+        text: "Teaching and Learning"
+      },
+      {
+        text: "5"
+      }
+    ],
+    [
+      {
+        text: "Mechanical Engineering"
+      },
+      {
+        text: "5"
+      }
+    ],
+    [
+      {
+        text: "English Language and Linguistics"
+      },
+      {
+        text: "5"
+      }
+    ],
+    [
+      {
+        text: "Mathematics and Physics"
+      },
+      {
+        text: "5"
+      }
+    ]
+    
+  ]
+}) }}
+
+The free text inputs for subjects are largely a mix of subjects which are not listed in the fixed list (English, physical education), or degrees where the candidate has done a mix of subjects (Mathematics and physics).
+
+We’ll look at whether we’re able to add the common subjects that are not in the current degree subject list. We can do the same for the common dual subjects, but there are lots of dual subject combinations. -We need toto do analysis on these to work out what would be the best thing for Register to store.
```

Comment:
"-We need toto do analysis on these to work out what would be the best thing for Register to store." — typos